cluster/images/build-terminal.Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The Karmada Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the official ttyd image as the base
+FROM tsl0922/ttyd:1.7.7
+
+# Switch to root so we can install packages or make changes
+USER root
+
+# (Optional) Install packages if needed
+# For example, if you need bash on Alpine:
+#RUN apk update && apk add --no-cache bash
+#RUN apt-get update && apt-get install -y curl
+RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
+
+
+# Download and install kubectl
+RUN curl -LO https://dl.k8s.io/release/v1.25.0/bin/linux/amd64/kubectl \
+  && mv kubectl /usr/local/bin/kubectl \
+  && chmod +x /usr/local/bin/kubectl
+
+# Download and install karmadactl (version v1.3.0)
+RUN curl -L -o /tmp/karmadactl-linux-amd64.tgz \
+    https://github.com/karmada-io/karmada/releases/download/v1.3.0/karmadactl-linux-amd64.tgz \
```

Comment:
I think v1.3.0 cloud be a dockerfile args.  BTW the latest version of karmada is v1.13.2

https://github.com/karmada-io/karmada/releases
![image](https://github.com/user-attachments/assets/31f8900b-ae9c-4050-abf6-104720b05d4b)